modules/core/src/test/java/org/apache/ignite/testframework/junits/multijvm/IgniteNodeRunner.java

```diff
@@ -69,7 +75,12 @@ public static void main(String[] args) throws Exception {
 
         X.println("Starting Ignite Node... Args=" + Arrays.toString(args));
 
-        IgniteConfiguration cfg = readCfgFromFileAndDeleteFile(args[0]);
+        IgniteConfiguration cfg;
+
+        if (args.length > 1)
```

Comment:
It may have length 0 but will not be null in case of normal execution.
But I can't find such information in JLS: http://docs.oracle.com/javase/specs/jls/se7/html/jls-12.html